src/java/test/org/apache/zookeeper/test/ShedClientsTest.java

```diff
@@ -0,0 +1,161 @@
+/**
```

Comment:
This test uses a single server and client, which doesn't seem ideal to test probabilities.
Consider doing something similar to src/java/test/org/apache/zookeeper/test/StaticHostProviderTest.java
which was developed as part of ZK-1355 for testing zk.updatedServerList()